### PR TITLE
[blockzerolabs-cryptonauts] Blockzerolabs: Cryptonauts Strategy v2

### PR DESCRIPTION
**Pull request by umarsa**
Blockzerolabs: Cryptonauts Strategy v2

### CODE REVIEW

File: src/strategies/blockzerolabs-cryptonauts/index.ts [hunk 1]
```diff
@@ -21,70 +11,13 @@ export async function strategy(
...
-          totalBalance = totalBalance.add(vaultBalance[_nftId]);
-        }
-      }
-    }
-    // Format the balance with 18 decimals (fixed)
-    balances[address] = parseFloat(formatUnits(totalBalance, options.decimals));
   });
-
-  return balances;
+  return await response.json();
```
Comment (ChaituVR) [inline src/strategies/blockzerolabs-cryptonauts/index.ts:22]: @umarsa we don't need a separate strategy for this right? we can call the `api` strategy directly right? if API can be modified
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/api/index.ts

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/blockzerolabs-cryptonauts/index.ts (hunk 1 above)
Comment (umarsa) [reply src/strategies/blockzerolabs-cryptonauts/index.ts:22]: Yeah, we may be able to use that pre-existing strategy.

Do you know what the "addresses" field is sending over to the endpoint?
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/api/index.ts#L18

My concern is that the URL might become unnecessarily long and potentially break the RFC guidelines..

Comment (ChaituVR) [reply src/strategies/blockzerolabs-cryptonauts/index.ts:22]: Hmm yea maybe i think you can try https://snapshot.org/#/strategy/api-post where you can send all addresses in body instead, let me know if it works :)

**Comment by ChaituVR** [comment]
Closing this for https://github.com/snapshot-labs/snapshot-strategies/pull/193#discussion_r757075693 . feel free to reopen or create a new PR

**Pull request closed without merging**